### PR TITLE
Lead surface

### PR DESCRIPTION
**Pull request by benjag**
Adds surface support to lead container

### CODE REVIEW

File: src/components/lead/CdrLead.vue [hunk 1]
```diff
@@ -1,24 +1,40 @@
...
 });
 
-const style = useCssModule();
+interface leadProps {
+  /** 
+   * Sets the orientation of the background surface
+   * @demoSelectMultiple false
+   * @values 'top', 'bottom'
+   */
+  surfaceOrientation: 'top' | 'bottom'
```
Comment (mhewson) [inline src/components/lead/CdrLead.vue]: surfaceOrientation seems a bit of a mouthful would surface:top /bottom do the same?

File: src/components/lead/examples/Lead.vue [hunk 2]
```diff
@@ -10,14 +10,24 @@
...
+      <template #top>
         <CdrImg src="/src/dev/static/cedar-1920x1080.jpg" />
       </template>
-      <template #content>
+      <template #bottom>
         <CdrTitle>Global lead title</CdrTitle>
       </template>
     </CdrLead>
 
+    <h2>Lead container with surface on top</h2>
```
Comment (mhewson) [inline src/components/lead/examples/Lead.vue:21]: nice!

File: src/components/lead/styles/vars/CdrLead.vars.scss [hunk 3]
```diff
@@ -1,10 +1,18 @@
 @mixin cdr-lead-base-mixin() {
   width: 100%;
   line-height: 0;
-  background: linear-gradient(
-    transparent 0 50%,
-    var(--cdr-lead-base-background-color, var(--cdr-color-background-primary, #{$cdr-color-background-primary})) 26% 100%,
-  );
+  &--top {
+    background: linear-gradient(
+      var(--cdr-lead-base-background-color, var(--cdr-color-background-primary, #{$cdr-color-background-primary})) 0 50%,
```
Comment (mhewson) [inline src/components/lead/styles/vars/CdrLead.vars.scss]: is this the surface color - the background color should come from leads parent

**Review by mhewson — APPROVED** [review]
looks grand!

File: src/components/lead/styles/vars/CdrLead.vars.scss (hunk 3 above)
Comment (benjag) [reply src/components/lead/styles/vars/CdrLead.vars.scss]: Meaning the custom prop should be named `--cdr-lead-surface-color`? The tokens for surface don't exist so the fallbacks would still be background color right?

File: src/components/lead/styles/vars/CdrLead.vars.scss [hunk 4]
```diff
@@ -3,14 +3,14 @@
   line-height: 0;
   &--top {
     background: linear-gradient(
-      var(--cdr-lead-base-background-color, var(--cdr-color-background-primary, #{$cdr-color-background-primary})) 0 50%,
+      var(--cdr-lead-surface-color, var(--cdr-color-background-primary, #{$cdr-color-background-primary})) 0 50%,
```
Comment (mhewson) [inline src/components/lead/styles/vars/CdrLead.vars.scss:6]: I wonder if there is time to get 
--cdr-surface
--cdr-surface-recessed 
into tokens and then default the surface color to --cdr-surface - and allow the prop to switch its token?

Comment (benjag) [reply src/components/lead/styles/vars/CdrLead.vars.scss:6]: Can follow up with that, I want to get started on a round of integration testing

**Pull request merged**